dora/core/common/pom.xml

```diff
@@ -155,6 +155,12 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <version>${jupiter.version}</version>
```

Comment:
why is Jupiter included in this PR? The dependency is not used anywhere. Besides, we are still on JUnit 4, does Jypiter even work with JUnit 4?